version/2_4.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash -e
+
+echo "================= Installing Python 2.4.6 ==================="
+sudo add-apt-repository -y ppa:deadsnakes/ppa
+apt-get update
+sudo apt-get install -y \
+  python2.4=2.4* \
+  python-dev=2.7* \
```

Comment:
why is python-dev, virtualenv and python-pip being installed multiple times with same version?

Reply:
This are all the supported to ubuntu 14.04.

https://packages.ubuntu.com/trusty/python-pip
https://packages.ubuntu.com/trusty/python-dev
https://packages.ubuntu.com/trusty/python-virtualenv

Reply:
ok